src/actor-widget/actor-widget.ts

```diff
@@ -0,0 +1,299 @@
+import { GridOptions } from 'ag-grid-community';
+import { CrabService } from '../services/crab.api-service';
+import { autoinject } from 'aurelia-framework';
+import { bindable } from 'aurelia-templating';
+import { DialogController } from 'aurelia-dialog';
+
+@autoinject
+export class ActorWidget {
+  @bindable public scope: any;
+  @bindable public actorenApiService: any;
+  @bindable public dialogController: DialogController;
+
+  public showSpinner: boolean = true;
+  public gridOptions: GridOptions;
+  public zoekterm: string;
+  public selectedActor: any;
+  public showTable: boolean = true;
+  public showActor: boolean = false;
+  public showFilters: boolean = false;
+  public isAdvancedSearch: boolean = false;
+  public landen: any[] = [];
+  public gemeenten: any[] = [];
+  public postcodes: any[] = [];
+  public straten: any[] = [];
+  public huisnrs: any[] = [];
+  public suggest: any = {};
+
+  private filters: any = {};
+
+  constructor(private crabService: CrabService) {
+    this.loadLanden();
+    this.suggest.gemeenten = { suggest: value => this.loadGemeenten(value) };
+    this.suggest.postcode = { suggest: value => this.loadPostcodes(value) };
+    this.suggest.straten = { suggest: value => this.loadStraten(value) };
+    this.suggest.huisnummer = { suggest: value => this.loadHuisnrs(value) };
+  }
+
+  public bind() {
+    this.gridOptions = {} as GridOptions;
+    this.gridOptions.context = this;
+    this.gridOptions.enableColResize = true;
```

Comment:
enablecolresize geeft deprecated warning, gebruik resizable: true in defaultcoldefs